tests/components/forms/test_debounce.py

```diff
@@ -43,7 +43,7 @@ def on_change(self, v: str):
 
 def test_render_child_props():
     """DebounceInput should render props from child component."""
-    tag = rx.debounce_input(
+    tag: str = rx.debounce_input(
```

Comment:
```suggestion
    tag = rx.debounce_input(
```

the other similar changes in this file should also be reverted